### ISSUE TITLE
Output filename w/o special characters

### ISSUE THREAD

**Issue by tgruene** [epoc-ed/GUI#23]
The current output filenames contain a colon ':' as part of the
timestamp. colon is a special character for shells and thus need to be
escaped. This makes some programming cumbersome, e.g. for backup
processes, shell scripts, etc.

Can we revert to Erik's previous solution and use e.g. 
PPh4I3_000_D2024_05_29_T204958.h5 instead of
PPh4I3_000_29_May_2024_20:49:58_master.h5 ?

Or do you think that readability may be an issue?

**Comment by kferjaoui** [comment]
You are right! I have already fixed it (replaced with _) but didn't push it yet. Will do asap. Thanks
